// myexpress/app/controllers/mycontoller.js
   var mysql = require('mysql');
   var conn = mysql.createConnection(
   {
   	 host : 'localhost',
   	 user : 'root',
   	 password : 'root',
   	 database : 'test'
   });
   conn.connect();

module.exports.inserted=function(req, res) 
{
    var result = '';

    var quer = "INSERT INTO student values('"+req.query.first_name+"','"+req.query.last_name+"',"+req.query.roll+");";
    console.log(quer);
    conn.query(quer, function(err, rows, fields) {
   	if(!err)
   	    {
   	    	
   		   console.log(rows);
   		   res.send("inserted "); 
   		}
   	else
   	    {
   		 //res.send('Error!!!');
   	     console.log('error!!!!');
   	    }	
   	   
   });

  // conn.end();
}

module.exports.getalldata=function(req, res)
{

   //console.log(conn.connect());
   conn.query('SELECT * from student', function(err, rows, fields) {
   	if(!err)
   	    {
   	    	var out="";
           for(i in rows)
           {  
   		    console.log('The Solution is ' + rows[i].Name);
   		    out += rows[i].Name+" "+rows[i].Dept+" " +rows[i].Rollno +"<br>";
   		   } 
   		   res.send('The Solution is: <br> ' + out);
   		}
   	else
   	    {
   		 res.send('Error!!!');
   	     console.log('error!!!!');
   	    }	
   });

  //conn.end();
}
